modules/jt-examples-of-common-use-cases.adoc

```diff
@@ -0,0 +1,169 @@
+:_content-type: CONCEPT
+// Module included in the following assembly:
+//
+// jenkins-tekton/migrating-from-jenkins-to-tekton.adoc
+
+[id="jt-examples-of-common-use-cases_{context}"]
+= Examples of common use cases
+
+Both Jenkins and Tekton offer capabilities for common CI/CD use cases, such as: 
+
+* Compiling, building images, and deploying images
+* Extending the core capabilities by using plugins
+* Reusing shareable libraries and custom scripts
+
+This section briefly describes the comparable workflows for some common use cases in Jenkins and Tekton.
+
+== Running a maven pipeline in Jenkins and Tekton
+
+Maven is used extensively in the Jenkins workflows. You can map your existing workflow to Tekton by using the following examples:
+
+.Example: Compile, build an image, and deploy to OpenShift using maven on Jenkins
```

Comment:
How about, `Example: Compile and build an image and deploy it to OpenShift using maven on Jenkins` 

The original sentence construction makes me think I'm doing three separate actions: 1) compiling (not sure what), 2) building an image, 3) deploying the image to OpenShift.